bananas_server/web_routes.py

```diff
@@ -109,6 +171,17 @@ async def healthz_handler(request):
     return web.HTTPOk()
 
 
+@routes.get("/")
+async def root(request):
+    if (
+        "websocket" == request.headers.get("Upgrade", "").lower().strip()
+        and "upgrade" in request.headers.get("Connection", "").lower()
+    ):
```

Comment:
this is ugly enough that it might be worth splitting them out into separate variables to make the line shorter (and call strip() consistently?)